### PR TITLE
Add setup and quickstart guide to the package documentation

### LINKED ISSUE #6720: Update documentation of the `ballerinax/stripe` connector
**Description:**
> $subject

Following components should be included:
- [x] Readme with proper package overview and build instructions
- [x] Setup guide
- [x] Quickstart guide

### PR DESCRIPTION
**Pull request by ayeshLK**
Fixes: [#6720](https://github.com/ballerina-platform/ballerina-library/issues/6720)

## Examples
N/A

## Checklist
- [X] Linked to an issue
- [ ] ~Updated the changelog~
- [ ] ~Added tests~
- [ ] ~Updated the spec~
- [ ] ~Checked native-image compatibility~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,88 @@
...
+[Stripe](https://stripe.com/) is a leading online payment processing platform that simplifies the handling of financial transactions over the Internet. Stripe is renowned for its ease of integration, comprehensive documentation, and robust API that supports a wide range of payment operations including credit card transactions, subscription management, and direct payouts to user bank accounts.
 
-The Stripe API is organized around REST. Our API has predictable resource-oriented URLs, accepts form-encoded request bodies, returns JSON-encoded responses, and uses standard HTTP response codes, authentication, and verbs.
+The Ballerina Stripe Connector allows developers to interact with the [Stripe REST API](https://stripe.com/docs/api), enabling seamless integration of Stripe’s extensive payment processing capabilities into Ballerina applications. This connector facilitates the automation of various payment-related operations such as charge creation, customer management, billing, and direct payouts. By leveraging the Ballerina Stripe Connector, developers can build secure and scalable payment solutions that enhance the e-commerce capabilities of their applications.
```
Comment (NipunaRanasinghe) [inline README.md]: Let's mention the supported REST API version

File: ballerina/Package.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+## Package overview
```
Comment (NipunaRanasinghe) [inline ballerina/Package.md]:
```suggestion
## Overview
```

File: README.md (hunk 1 above)
Comment (ayeshLK) [reply README.md]: There is no difference in the REST API version AFAIU. They only have V1.

File: ballerina/Package.md (hunk 2 above)
Comment (ayeshLK) [reply ballerina/Package.md]: In the `package.md` the usual practice is to use `Package overview`. Do we need to change that ?

File: README.md (hunk 1 above)
Comment (NipunaRanasinghe) [reply README.md]: Still better to mention the REST API version to be future proof IMO (e.g. incase they introduce a new API version in near future)

Comment (ayeshLK) [reply README.md]: Fixed with [6b9f281010b00f8a81f1fe10af1b5e1c98b57ddf](https://github.com/ballerina-platform/module-ballerinax-stripe/pull/3/commits/6b9f281010b00f8a81f1fe10af1b5e1c98b57ddf)

File: ballerina/Package.md (hunk 2 above)
Comment (ayeshLK) [reply ballerina/Package.md]: Fixed with [6b9f281010b00f8a81f1fe10af1b5e1c98b57ddf](https://github.com/ballerina-platform/module-ballerinax-stripe/pull/3/commits/6b9f281010b00f8a81f1fe10af1b5e1c98b57ddf)

**Review by NipunaRanasinghe — DISMISSED** [review]
LGTM

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**